nab/detectors/htmjava/htmjava_detector.py

```diff
@@ -89,11 +89,12 @@ def handleRecord(self, inputData):
       self.minVal = value
 
     # Compute log(anomaly likelihood)
-    anomalyScore = self.anomalyLikelihood.anomalyProbability(
-      inputData["value"], rawScore, inputData["timestamp"])
-    logScore = self.anomalyLikelihood.computeLogLikelihood(anomalyScore)
     if spatialAnomaly:
       logScore = 1.0
+    else:
```

Comment:
@lscheinkman The call to anomalyProbability updates internal estimates of the anomaly likelihood. We should call it even in the case of spatial anomalies.